services/spar/src/Spar/Scim/User.hs

```diff
@@ -609,12 +627,15 @@ getOrCreateScimUser stiTeam brigUser = do
       handle <- getUserHandle' brigUser'
       let name = userDisplayName brigUser'
       richInfo <- getRichInfo' uid
+      status <- getStatus' uid
       -- NOTE: If user is not an SSO User; this returns Nothing
       -- Hence; we should only set managedByScim if this _succeeds_
       ssoIdentity' <- getSSOIdentity' brigUser'
       externalId <- toExternalId' ssoIdentity'
       setManagedBy' uid ManagedByScim
-      let neededInfo = NeededInfo handle name externalId richInfo
+      -- NOTE: A user can be 'Active | Deleted | Ephemeral | Suspended'. We
+      -- only consider them Active when they're 'Active'
+      let neededInfo = NeededInfo handle name externalId richInfo (Just (status == Active))
```

Comment:
what about ephemeral users?  can we even create them here?  but even if not, we're moving towards accessing brig data directly from both brig and scim, and this will be an interesting case.

Reply:
e86c8c13c77a6c0633d8fa6b4af687619455b691 94168e7a7a6c1626d89b9b34d45d0f608291b614

I think this is much clearer, and it handles deleted and ephemeral users more robustly.